README.md

```diff
@@ -1,2 +1,11 @@
 # virtio-bindings
-Virtual IO (virtio) kernel bindings
+Rust FFI bindings to virtio generated using [bindgen](https://crates.io/crates/bindgen).
+
+# Usage
+Add this to your `Cargo.toml`:
+```toml
+virtio-bindings = { version = "0.1", features = ["virtio-v5_0_0"]}
```

Comment:
We should be able to add:

```
virtio-bindings = { version = "0.1"}
```

and this should use the latest version by default (5.0).

Then we should document the default behavior and how to use a different version than the default one.